catroidTest/src/org/catrobat/catroid/uitest/util/UiTestUtils.java

```diff
@@ -2132,4 +2133,12 @@ public boolean isSatisfied() {
 
 		return wantedState;
 	}
+
+	public static void leaveStageSafeButSlow(Solo solo) {
+		if(solo.getCurrentActivity().getClass() == StageActivity.class){
```

Comment:
Missing space, also, do you want to compare like this instead of using `instanceof`? Just checking.